### DISCUSSION TITLE
LSP4IJ 0.9.0 Release

### DISCUSSION THREAD

**Discussion by angelozerr** [redhat-developer/lsp4ij#682] [Announcements]
Hi,

As I have introducted [the bug ](https://github.com/redhat-developer/lsp4ij/issues/660) in 0.8.1 with project indexing support (start language servers only when all project are indexed to provide the best performance on IJ startup), we are about to release LSP4IJ 0.9.0. Highlights of this release include 

 * [the bug ](https://github.com/redhat-developer/lsp4ij/issues/660) in 0.8.1
 * [LSP selectionRange](https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPSupport.md#selection-range)
 * Use standard IJ menu for Implementations / Type declarations
 * Improve LSP folding range with IJ codeblock support 
 * Improve LSP completion to hide completion from default IJ word completion 
 * Show workspace symbol with Class, Struct, Type kind in the standard Class tab (Ctrl+N)
 * fixing several bugs 

The release will integrate also https://github.com/redhat-developer/lsp4ij/pull/680 once the PR will be merged
 
More details in the <a href="https://github.com/redhat-developer/lsp4ij/milestone/9?closed=1">changelog</a>.
 
Please give a try to the [latest nightly build](https://plugins.jetbrains.com/plugin/23257-lsp4ij/versions/nightly).

Unless you report a blocking issue, we'll proceed with the release next Friday 12/13/2024.

Thanks so much to @SCWells72 with his great contributions!
 
 cc @CppCXY @InSyncWithFoo @seachicken @KUGDev @AlexWeinstein92 @aparnamichael @MituuZ @turkeylurkey @mrglavas @koxudaxi @FalsePattern @onriv @kristofvb @kdvolder @rcjsuen @jansorg @SCWells72

**Comment by dessina-devasia** [comment]
@angelozerr Considering the null pointer exception issue encountered while opening projects in the latest nightly build (https://github.com/redhat-developer/lsp4ij/issues/696), can we expect another nightly build before the release?

**Reply by angelozerr** [reply]
Yes sure, it should be available.

**Comment by angelozerr** [comment]
LSP4IJ 0.9.0 has been released and are now available in marketplace https://plugins.jetbrains.com/plugin/23257-lsp4ij/edit/versions/stable/651509
